### DISCUSSION TITLE
ERR_SSL_PROTOCOL_ERROR when accessing new Nextcoud (Tailscale tailnet vpn setup)

### DISCUSSION THREAD

**Discussion by idealemu** [nextcloud/all-in-one#4848] [Questions] [answered]
Hi,
I have just setup AIO, following the reverse proxy instructions, tried all the debugging steps but cannot get a connection to work to my server.
I have a domain set to point to an IP on my tailnet, tailscale VPN, only machine on this network can reach this IP. I can access the "containers" page using either `myTailnetIP:8080/containers` or `myServersLANIP:8080/containers`.  The domain pointing to my tailscale IP was set up, but obviously (after trying every other step) I have to enable skip verification.

This is my Caddyfile:
```
  https://myDomain:443 {
      header Strict-Transport-Security max-age=31536000;
      reverse_proxy localhost:11000
  }
   
 https://myDomain:8443 {
      reverse_proxy https://localhost:8080 {
          transport http {
              tls_insecure_skip_verify
          }
      }
  }
  ```

My `nextcloud-aio-apache` logs:
```
Waiting for Nextcloud to start...
Connection to nextcloud-aio-nextcloud (172.19.0.8) 9000 port [tcp/*] succeeded!
{"level":"info","ts":1718717708.068626,"msg":"using config from file","file":"/tmp/Caddyfile"}
{"level":"info","ts":1718717708.0837615,"msg":"adapted config to JSON","adapter":"caddyfile"}
[Tue Jun 18 13:35:08.350660 2024] [mpm_event:notice] [pid 530:tid 281473602620672] AH00489: Apache/2.4.59 (Unix) configured -- resuming normal operations
[Tue Jun 18 13:35:08.350861 2024] [core:notice] [pid 530:tid 281473602620672] AH00094: Command line: '/usr/local/apache2/bin/httpd -D FOREGROUND'
Connection to nextcloud-aio-nextcloud (172.19.0.8) 9000 port [tcp/*] succeeded!
[Wed Jun 19 13:14:51.749744 2024] [mpm_event:notice] [pid 34:tid 281473080519936] AH00489: Apache/2.4.59 (Unix) configured -- resuming normal operations
[Wed Jun 19 13:14:51.749900 2024] [core:notice] [pid 34:tid 281473080519936] AH00094: Command line: '/usr/local/apache2/bin/httpd -D FOREGROUND'
{"level":"info","ts":1718802891.7787194,"msg":"using config from file","file":"/tmp/Caddyfile"}
{"level":"info","ts":1718802891.7885513,"msg":"adapted config to JSON","adapter":"caddyfile"}
```
I tried accessing my Nextcloud instance through both `https://myIPOrDomain` or `https://myIPOrDomain:11000` but both give me a `ERR_SSL_PROTOCOL_ERROR`.

I am using tailscale because my ISP doesn't offer IPV4 static addresses. My tailnet has two machines, the server, and my laptop trying to access Nextcloud. I can successfully use the tailnet to ssh into the server, and I can view the containers page using the tailscale IP, so I *assume* nothing is wrong there. (There might be something to do with `tailscale serve` to open ports, but I am not quite sure if it is appropriate for this case.)

All containers are successfully running.

Any help is appreciated !

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/blob/main/local-instance.md
